Resources/Prototypes/DeltaV/Entities/Clothing/OuterClothing/coats.yml

```diff
@@ -119,3 +119,17 @@
   - type: TemperatureProtection
     coefficient: 0.1
 
+- type: entity
+  parent: ClothingOuterStorageFoldableBase
```

Comment:
Perhaps this should be parented to Chemist's Winter Coat and inherit its protection stats, if it has any?

Reply:
The protection stats are already there, and the parent needs to be foldable, like any lab coat.